pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -394,6 +406,24 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {"class": "reduce", "mode": self.mode, "axes": self.axes, "from": self._get_input_layer_name(input)}
 
 
+class Length(Module):
+  """
+  Wraps RETURNN LengthLayer.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, axis: Union[str, Dim, int]):
```

Comment:
Now that you implemented `originating_tensor_axis`, I think to keep it consistent, we should not mix RETURNN stuff here in the PyTorch API, and only allow `int` here.